src/app/components/challenge/challengeparticipate/challengeparticipate.component.html

```diff
@@ -1,10 +1,17 @@
-<div class="challenge-participate-container">
-  <div class="inactive" *ngIf="!isActive">
-    Sorry, The Challenge is not active!
+<div class="challenge-card">
+  <div *ngIf="!isActive" class="ev-card-panel ev-md-container card-bt-margin">
```

Comment:
Behaviorally when I'm logged out and try to load a page we are first seeing `Sorry, The Challenge is not active!` message and then switching to `Please Login to access this section.` Ideally we shouldn't be seeing the former message if we are not logged-in in first place.

Reply:
Thanks for creating an issue @Sanji515 :https://github.com/Cloud-CV/EvalAI-ngx/issues/181